lib/KeyboardAwareMixin.js

```diff
@@ -76,7 +78,7 @@ const KeyboardAwareMixin = {
   },
 
   resetKeyboardSpace: function () {
-    const keyboardSpace: number = (this.props.viewIsInsideTabBar) ? _KAM_DEFAULT_TAB_BAR_HEIGHT + this.props.extraScrollHeight : this.props.extraScrollHeight
+    const keyboardSpace: number = (this.props.viewIsInsideTabBar) ? _KAM_DEFAULT_TAB_BAR_HEIGHT : 0 
```

Comment:
Please add support for the `extraScrollHeight` prop. In your changes this prop is missing.